.circleci/config.yml

```diff
@@ -45,6 +45,43 @@ references:
       key: v1-npm-{{ arch }}-{{ checksum "calypso/.nvmrc" }}-{{ checksum "package-lock.json" }}-{{ checksum "calypso/npm-shrinkwrap.json" }}
       paths:
         - ~/.npm
+  app_cache_paths: &app_cache_paths
+    - calypso-hash
+    - build
+    - resource/certificates
+    - calypso/public
+    - calypso/server
+    - calypso/config
+    - calypso/package.json
+    - calypso/npm-shrinkwrap.json
+    - calypso/.nvmrc
+  calypso_prepare_cache: &calypso_prepare_cache
+    run:
+      name: Prepare calypso cache
+      command: |
+              if [ -n "${CALYPSO_HASH}" ]; then
+                # cache bust e2e tests
+                uuidgen > calypso-current-hash
```

Comment:
Why use a unique ID not depending on the Calypso hash? Couldn't we just `echo "${CALYPSO_HASH}" > calypso-current-hash`?